README.md

```diff
@@ -1,52 +1,226 @@
 
-# reshuffle-_CONNECTOR_NAME_-connector
+The following example exposes an endpoint to return the number of items in the external collection.
+It uses a postgres database configured with a Reshuffle pgsql connector
 
-### Reshuffle _CONNECTOR_NAME_ Connector
+```js
+const { Reshuffle } = require('reshuffle')
+const { WixConnector } = require('@reshuffle/wix-connector')
+const { PgsqlConnector } = require('reshuffle-pgsql-connector')
 
-This connector provides <description>.
+const app = new Reshuffle()
+
+// The collection (or table) name
+const COLLECTION = 'tasks'
+
+const pg = new PgsqlConnector(app, {
+  url: process.env.WIX_DB_URL
+})
+
+const wix = new WixConnector(app, {
+  secret: process.env.RESHUFFLE_WIX_SECRET,
+})
+
+wix.on({ action: 'data/count' }, async (event, app) => {
+  const { collectionName, filter } = event.request.body
+  if (collectionName === COLLECTION) {
+    const todos = await pg.query(`SELECT * from ${COLLECTION}`)
+    event.response.status(200).json({ totalCount: todos.rowCount })
+  } else {
+    event.response.status(400).json({ 'message': 'Bad request. We only have todos' })
+  }
+})
+
+app.start()
+```
+
+### Table of Contents
+
+[Configuration Options](#configuration)
+
+[Events](#events)
+
+[Actions](#actions)
+
+[Utilitiey Functions](#utils)
+
+### <a name="configuration"></a> Configuration options
+
+```js
+const app = new Reshuffle()
+const wix = new WixConnector(app, {
+  secret: process.env.RESHUFFLE_WIX_SECRET,
+  webhookPath: process.env.RESHUFFLE_WIX_WEBHOOK,
+})
+```
+Both `secret` and `webhookPath` are optional.
+
+The secret is a string containing a shared secret as described in [Wix Authentication](https://www.wix.com/corvid/reference/spis/external-database-collections/external-database-collections/authentication)
+
+When configuring the external database collection on Wix, you can define a `settings` object that is sent from Wix to the connector with every request.
+
+To use (the optional) secret, you should configure the settings object to include:
+```json
+{
+  "secret": ".....your secret....."
+}
+```
+Wix then includes it in the incoming request like so:
+```json
+{
+    "requestContext": {
+        "settings": {
+            "secret": ".....your secret....."
+        },
+        "instanceId": "...",
+        "installationId": "...",
+        "memberId": "...",
+        "role": "..."
+  }
+}
+``` 
+If you do not define a secret in the connector configuration, then it will not expect one.
+
+You can use the `webhookPath` to configure the url that Wix hits when it makes its calls to
+your external database collection. The value of `webhookPath` will be appended to your runtime's
+base url.
+Left unprovided, `webhookPath` defaults to `/webhooks/wix`, so if your runtime runs at `https://example.com/` then
+your full webhook path is `https://example.com/webhooks/wix`. This is the path you'll need
+to register with Wix when defining a new external collection.
+
```

Comment:
Steps on how to register webhook in Wix would be great

Reply:
Added :)